stdlib/3.4/enum.pyi

```diff
@@ -45,11 +45,11 @@ if sys.version_info >= (3, 6):
         def __xor__(self: _T, other: _T) -> _T: ...
         def __invert__(self: _T) -> _T: ...
 
-    # All `type: ignore` comments below due to IntFlag making the function signatures more permissive.
+    # int and Flag have some incompatible methods
```

Comment:
This is unclear -- is the incompatibility between int and Flag, or between Flag and IntFlag? I liked the old comment better in that it makes it clear that IntFlag has more relaxed argument types (which is fine, it's not an incompatibility, otherwise mypy would complain).